src/server/saved_objects/client/saved_objects_client.js

```diff
@@ -74,6 +101,45 @@ export class SavedObjectsClient {
     };
   }
 
+  /**
+   * Returns an array of objects by id
+   *
+   * @param {array} ids - an array ids, or an array of objects containing id and optionally type
+   * @param {string} type - applies type to each id
+   * @returns {promise} Returns promise containing array of documents
+   * @example
+   *
+   * // single type:
+   * bulkGet(['one', 'two', 'config'])
+   *
+   * // mixed types:
+   * bulkGet([
+   *   { id: 'one', type: 'config' },
+   *   { id: 'foo', type: 'index-pattern'
+   * ])
+   */
+  bulkGet(ids = [], type) {
```

Comment:
Also, please follow the pattern of using `async`/`await` like the rest of the methods.